api-samples/tabCapture/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Tab Capture Example",
+  "description": "Capture a tab and play in a <video> element in a separate tab.",
+  "version": "1",
+  "manifest_version": 3,
+  "action": {
+    "default_icon": "icon.png"
+  },
+  "background": {
+    "service_worker": "background.js"
```

```suggestion
    "service_worker": "service-worker.js"
```
Can we switch this file name to service-worker.js? Thanks 😄